README.md

```diff
@@ -4,13 +4,19 @@ SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 
 # beman.exemplar: A Beman Library Exemplar
 
-![Continuous Integration Tests](https://github.com/bemanproject/exemplar/actions/workflows/ci_tests.yml/badge.svg)
+<!-- markdownlint-disable -->
+<img src="https://github.com/bemanproject/beman/blob/main/images/logos/beman_logo-beman_library_under_development.png" style="width:5%; height:auto;"> ![Continuous Integration Tests](https://github.com/bemanproject/exemplar/actions/workflows/ci_tests.yml/badge.svg)
+<!-- markdownlint-enable -->
```

Comment:
Is there a reason to use `<img>` html tag instead of markdown facility here?

Also, can we put this two in separate lines?

Reply:
According to markdown-lint documentation.

> Enable all rules: <!-- markdownlint-enable -->

This will cause the previously configured ignore rule to be flushed out.

Is there a reason we need to use this disable-enable pair?

Why not use:
```suggestion
<!-- markdownlint-disable-next-line -->
<img src="https://github.com/bemanproject/beman/blob/main/images/logos/beman_logo-beman_library_under_development.png" style="width:5%; height:auto;">
![Continuous Integration Tests](https://github.com/bemanproject/exemplar/actions/workflows/ci_tests.yml/badge.svg)
```

Reply:
Why not simply do:

```suggestion
![Library status](https://github.com/bemanproject/beman/blob/main/images/logos/beman_logo-beman_library_under_development.png)
![Continuous Integration Tests](https://github.com/bemanproject/exemplar/actions/workflows/ci_tests.yml/badge.svg)
```

Reply:
Second syntax for including image doest not allow resize, at least not in standard Markdown on GitHub. You can simulate a change by editing a file in UI and do preview.